querybook/webapp/components/QueryRunButton/QueryRunButton.tsx

```diff
@@ -249,3 +268,60 @@ const QueryLimitSelector: React.FC<{
         </Dropdown>
     );
 };
+
+const TableSamplingConfig = PublicConfig.table_sampling ?? {
+    enabled: false,
+    sample_rates: [],
+    default_sample_rate: 0,
+};
+const sampleRateOptions = [{ label: 'none', value: 0 }].concat(
+    TableSamplingConfig.sample_rates.map((value) => ({
+        label: value + '%',
+        value,
+    }))
+);
+const DEFAULT_SAMPLE_RATE = TableSamplingConfig.default_sample_rate;
+const TableSamplingSelector: React.FC<{
```

Comment:
move to another component file

Reply:
will address it in a separate PR to use backend limit logic.